frontend/views/containers/group-settings/GroupRulesSettings.vue

```diff
@@ -0,0 +1,162 @@
+          ) Change
+
+      banner-simple.c-banner(
+        severity='info'
+        data-test='ruleAdjusted'
+        v-if='isVotingRuleAdjusted(rule)'
+      )
+        i18n * This value was automatically adjusted because your group is currently smaller than the disagreement number.
+
+      i18n.link(
+        v-if='!isRuleActive(rule)'
+        tag='button'
+        @click='openVotingProposal(rule)'
+      ) Change to this voting system
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import { mapGetters } from 'vuex'
+import { RULE_PERCENTAGE, RULE_DISAGREEMENT } from '@model/contracts/voting/rules.js'
+import { OPEN_MODAL } from '@utils/events.js'
+import L from '@view-utils/translations.js'
+import BannerSimple from '@components/banners/BannerSimple.vue'
+
+export default {
+  name: 'GroupVotingSystem',
+  components: {
+    BannerSimple
+  },
+  data: () => ({
+    config: {
+      [RULE_PERCENTAGE]: {
+        title: L('Percentage based'),
+        explanation: L('Proposals are accepted when the required percentage of members agree to the proposal.'),
+        status: L('Percentage of members that need to agree:')
+      },
+      [RULE_DISAGREEMENT]: {
+        title: L('Disagreement number'),
+        explanation: L('Proposals are rejected when a certain number of members disagree with the proposal.'),
+        status: L('Maximum number of "no" votes:')
+      }
+    }
+  }),
+  computed: {
+    ...mapGetters([
+      'groupMembersCount',
+      'groupVotingRule'
+    ]),
+    votingRulesSorted () {
+      return this.groupVotingRule.rule === RULE_DISAGREEMENT ? [RULE_DISAGREEMENT, RULE_PERCENTAGE] : [RULE_PERCENTAGE, RULE_DISAGREEMENT]
+    },
+    votingRuleSettings () {
+      return this.groupVotingRule.ruleSettings[this.groupVotingRule.rule]
+    }
+  },
+  methods: {
+    openVotingProposal (rule) {
+      sbp('okTurtles.events/emit', OPEN_MODAL, 'ChangeVotingRules', { rule })
+    },
+    isRuleActive (rule) {
+      return rule === this.groupVotingRule.rule
+    },
+    isVotingRuleAdjusted (ruleName) {
+      if (!this.isRuleActive(ruleName) || ruleName !== RULE_DISAGREEMENT) return false
+
+      return this.votingRuleSettings.threshold !== this.votingRuleSettings.thresholdAdjusted
+    },
+    votingValue (option) {
+      const HTMLTags = {
+        b_: '<span class="has-text-bold">',
+        _b: '</span>',
+        sm_: '<span class="has-text-1 has-text-small">',
+        _sm: '</span>'
+      }
+
+      if (option === RULE_PERCENTAGE) {
+        const percent = this.votingRuleSettings.threshold * 100 + '%'
+
+        // TODO/BUG/REVIEW: in a 1 member group with 49%, it says: 0 out of 1 members. Is it rights?
+        return L('{b_}{percent}{_b} {sm_}({count} out of {total} members){_sm}', {
+          percent,
+          count: Math.round(this.groupMembersCount * this.votingRuleSettings.threshold),
+          total: this.groupMembersCount,
+          ...HTMLTags
+        })
+      } else {
```

Comment:
`else if`, always check the rule type explicitly, don't assume there will only be two voting rule types since that's unlikely to be the case. The `else` case should show an error telling the user that they're probably using outdated software.

Reply:
Gotcha. Updating code.